ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -105,6 +105,44 @@ Item {
                                                     tokenAddress)
     }
 
+    // This function is called once `1:1` or `group` chat is created.
+    function checkForCreateChatOptions(chatId) {
+        if(root.rootStore.createChatStartSendTransactionProcess) {
```

Comment:
I am not sure how startSendTransaction is part of checkForCreatedChatOptions?
Because in this case, chat would already be created and we are only launching the transaction flow? 
Maybe we handle this in a separate function like openSendTransactionPopup etc?